Sources/App/Extensions/Security.swift

```diff
@@ -0,0 +1,59 @@
+//
+// This source file is part of the ENGAGE-HF-AI-Voice open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import Security
+
+enum SecKeyError: Error {
+    case invalidBase64Data
+    case unableToCreate
+    case unableToDecrypt
+}
+
+extension SecKey {
+    static func fromPEM(_ pem: String) throws -> SecKey {
+        // Strip headers/footers and whitespace.
+        let lines = pem
+            .components(separatedBy: .newlines)
+            .map { $0.trimmingCharacters(in: .whitespacesAndNewlines) }
+            .filter { !$0.hasPrefix("-----BEGIN") && !$0.hasPrefix("-----END") }
```

Comment:
Just to be sure that this also works when the key is injected as a long string in the env.

Reply:
Sounds good; thanks for checking this 👍